operator/v1/types_authentication.go

```diff
@@ -29,7 +29,17 @@ type AuthenticationStatus struct {
 	// Note that this field will be removed in the future releases, once https://github.com/openshift/enhancements/blob/master/enhancements/authentication/separate-oauth-resources.md is fully implemented
 	// +optional
 	ManagingOAuthAPIServer bool `json:"managingOAuthAPIServer,omitempty"`
-	OperatorStatus         `json:",inline"`
+
+	// LatestAvailableRevision is the latest revision used as suffix of revisioned
+	// secrets like encryption-config. A new revision causes a new deployment of pods.
+	//
+	// Usually, fields like that can only be used by one operand at a given time.
```

Comment:
why the extra comment?

Reply:
Oh, because oauth-proxy might need it to some day? Should we start putting it under an apiserver field?

Reply:
because that field can only be used by one `operand` at a time. Since `CAO` will manage two I wanted to highlight it in the comment not in the field's name.

Reply:
It makes me think whether we should treat the revisions similarly to the `generations` (that being an array), in case we would also like to use revisions for the other payload in the operator.

Reply:
where is an apiserver field?

Reply:
yeah, an array would be fine, the question is do you think `CAO` will always manage two operands? Maybe this is just a temporary situation and in the future we will split it?

Reply:
`status.oauthAPIserver. latestAvailableRevision`

Reply:
no reason this is temporary. And not reason an operator shouldn't have multiple operands.

Reply:
if it's permanent then `status.oauthAPIserver` gives us more flexibility in the future, @stlaz are you okay with that?

Reply:
The oauth server and oauth apiserver should indeed be coupled together, so this is indeed permanent.

The change seems fine